### ISSUE TITLE
Touchpoints wrapper should be an HTML button element

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4050]
## Area or Component
Conduct Test

## Quick Summary
The touchpoints wrapper is labelled as a `<div>`, but it's really a button (the fact that it isn't makes it not interactive for screenreader users)

## Accessibility Violation
* aria-role-missing
* The element's role is missing or is not appropriate for the element's function.

## Action Requested
* Make the touchpoints wrapper a `<button>` instead of a `<div>`

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/34cd6da49e112854b1c4a0832354e816b228fbd7/frontend/src/app/analytics/TouchpointsButton.tsx
